kobotoolbox/kpi
# feat(organizations): account sidenav update for MMOs TASK-977

### 🗒️ Checklist

1. [x] run linter locally
2. [x] update all related docs (API, README, inline, etc.), if any
3. [x] draft PR with a title `<type>(<scope>)<!>: <title> TASK-1234`
4. [x] tag PR: at least `frontend` or `backend` unless it's global
5. [x] fill in the template below and delete template comments
6. [x] review thyself: read the diff and repro the preview as written
7. [x] open PR & confirm that CI passes
8. [x] request reviewers, if needed
9. [ ] delete this section before merging

### 📣 Summary
Handle MMO and non-MMO orgs, as well as org roles, in the account sidebar.

### 👀 Preview steps
Based on figma designs [here](https://www.figma.com/design/VIn6Fv2oEpqDHaGoqs6Zt9/%E2%9A%99%EF%B8%8F-KPI-%2F-Organizations-%26-Teams-Feature?node-id=1732-41685&t=XhwXHn8qEGw7X4Oi-4)

Requires MMO feature flag

1. Create a user and check to ensure account sidenav remains unchanged on Stripe-enabled and disabled environments. The should be no changes for users who are not members of an MMO.
2. Make user's org an MMO, with them as an owner, and check the sidenav with Stripe enabled and disabled against the Figma designs for org owners.
3. Create another user and add them to the MMO (make sure to delete their personal org). Make them an admin and check against the figma designs as above.
4. Make second user a member (uncheck "admin" on their org user object in django admin) and check against the figma designs as above.
5. Toggle the use_team_label option in constance and see organization/team label change accordingly in sidenav.
